### ISSUE TITLE
An issue in CyclicGen_train_stage1.py

### ISSUE THREAD

**Issue by YaoooLiang** [alex04072000/CyclicGen#2]
hi, @alex04072000 
Loss was may forgotted to be calculated in CyclicGen_train_stage1.py.
`
            prediction1, flow1 = model1.inference(tf.concat([input1, input3, edge_1, edge_3], 3))
`

`total_loss = prediction1`

**Comment by alex04072000** [comment]
@YaoooLiang you are right, I just added the following code:
`reproduction_loss1 = model1.l1loss(prediction1, input2)`
`total_loss = reproduction_loss`

Thank you for your report!
